frontends/web/src/components/transactions/transaction.tsx

```diff
@@ -14,22 +14,53 @@
  * limitations under the License.
  */
 
-import { Component, h } from 'preact';
-import { translate } from 'react-i18next';
-import { FiatConversion } from '../rates/rates';
+import { Component, h, RenderableProps } from 'preact';
+import { translate, TranslateProps } from '../../decorators/translate';
+import { Fee } from '../../routes/account/send/send';
 import A from '../anchor/anchor';
-import * as style from './transaction.css';
-import * as parentStyle from './transactions.css';
 import { Dialog } from '../dialog/dialog';
 import { ProgressRing } from '../progressRing/progressRing';
+import { FiatConversion } from '../rates/rates';
+import { Amount } from '../rates/rates';
```

Comment:
Also import an interface here, isn't there a naming convention to see that it is an interface, or would you duplicate the interface in this file?

Reply:
i don't know about conventions, would also be interested. lmk if you find out :)

in typescript i don't even know exactly what the difference is between a type and an interface...

anyway, duplication no good :D

Reply:
In other places there is a similar situation and then the name is `ComponentnameInterface`
https://github.com/digitalbitbox/bitbox-wallet-app/blob/38d58be846abea9f9d2264df86c3024b0b3c4a9f/frontends/web/src/routes/account/account.tsx#L35

I propose to change it to `AmountInterface` in this case.

Reply:
random search result on the internet says:

> In TypeScript, type does not create a new name for instance. In TypeScript, an interface can create the new name that can be used everywhere. ... An interface can extend multiple interfaces and class as well. Type is mainly used when a union or tuple type needs to be used.